content/en/docs/reference/access-authn-authz/controlling-access.md

```diff
@@ -110,10 +110,11 @@ To learn more about Kubernetes authorization, including details about creating p
 
 Admission Control Modules are software modules that can modify or reject requests.
 In addition to the attributes available to Authorization Modules, Admission
-Control Modules can access the contents of the object that is being created or updated.
-They act on objects being created, deleted, updated or connected (proxy), but not reads.
-
-Multiple admission controllers can be configured.  Each is called in order.
+Control Modules can access the contents of the object that is being created or modified.
+They act on objects being created, deleted, modified or connected (proxy), but not reads. They can
+be applied by 4 verbs: create, delete, update, and connect. To clarify, here 'update' is plain English, not
```

Comment:
@MikeSpreitzer thanks a lot for the explanation! It's very comprehensive. I didn't catch all sources of confusion here since I'm new to k8s code base. 
For this PR, I'll just adopt what you suggest.